.github/renovate.json5

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  extends: ["config:base"],
+
+  "labels": ["dependencies"],
+
+  packageRules: [
+    // Check for updates, merge automatically
+    {
+      matchManagers: ["maven", "gradle", "gradle-wrapper"],
+      matchUpdateTypes: ["minor", "patch"],
+      automerge: true,
+      platformAutomerge: true
```

Comment:
nit: in json5 we should prefer trailing commas for object properties and list elements